tests/test.liq

```diff
@@ -79,6 +79,12 @@ def test.raise(fn) =
   try ignore(fn()) catch _ do () end
 end
 
+# Add a metric
+def test.metric(~category, ~name, ~value, ~unit)
+  data = "- name: \"#{(name:string)}\"\n  category: \"#{(category:string)}\"\n  value: #{(value:float)}\n  unit: \"#{(unit:string)}\"\n  time: #{(time():float)}\n\n"
+  file.write(append=true, data=data, "/tmp/metrics.yaml")
```

Comment:
I would suggest to use `./metrics.yaml` or something like that so that the file is located inside `_build/default/tests`

Reply:
I used to do that, but the tests can be run from various places so that an absolute path is much easier to handle...